admin-dev/themes/new-theme/js/pages/product/components/combination-list/bulk-choices-selector.ts

```diff
@@ -114,7 +114,7 @@ export default class BulkChoicesSelector {
         return;
       }
 
-      const isBulkSelectAll = checkbox.matches(`${CombinationMap.bulkSelectAll}, ${CombinationMap.bulkSelectAllInPage}`);
+      const isBulkSelectAll = checkbox.matches(CombinationMap.commonBulkAllSelector);
```

Comment:
this change actually not related to the fix, I just found while searching for the error cause and simpliefied it, it still works as inteded as both (`bulk select all` and `bulk select all in page`) buttons intentionally has that common class for that reason.